pkg/controller/disruption/disruption.go

```diff
@@ -994,6 +994,7 @@ func (dc *DisruptionController) updatePdbStatus(ctx context.Context, pdb *policy
 		DisruptionsAllowed: disruptionsAllowed,
 		DisruptedPods:      disruptedPods,
 		ObservedGeneration: pdb.Generation,
+		Conditions:         append([]metav1.Condition{}, pdb.Status.Conditions...),
```

Comment:
we can just assign the conditions from the copy, so we do not mutate the original pdb

Reply:
That's actually a pretty good idea, thank you! I tried to copy the array with the append.